test/codeu/chat/util/CommandTokenizerTest.java

```diff
@@ -0,0 +1,24 @@
+package codeu.chat.util;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class CommandTokenizerTest {
+
+	@Test
+	public void testTokenizer() {
+		CommandTokenizer quotesTokenizer = new CommandTokenizer(
+				"Command \"arg0\" \"arg1\" \"arg2\" \"arg3\"");
```

Comment:
Might want to include a test for this case, where two arguments have quotation marks but not spaces?

`Command "arg1""arg2"`

What is it supposed to do in this case? Adding a test will make it more obvious.